test/Chainweb/Test/Pact/RemotePactTest.hs

```diff
@@ -158,6 +163,31 @@ responseGolden networkIO rksIO = golden "command-0-resp" $ do
     return $! toS $! foldMap A.encode values
 
 
+sendValidationTest :: IO (Time Integer) -> IO ChainwebNetwork -> IO ()
+sendValidationTest iot nio = do
+    cenv <- fmap _getClientEnv nio
+    mv <- newMVar 0
+    SubmitBatch batch1 <- testBatch' iot 10000 mv
+    SubmitBatch batch2 <- testBatch' (return $ Time $ TimeSpan 0) 2 mv
+    let batch = SubmitBatch $ batch1 <> batch2
+    sid <- mkChainId version (0 :: Int)
+    expectSendFailure $ flip runClientM cenv $ do
+        sendApiCmd (testCmdsChainId sid) batch
+
+  where
+    expectSendFailure act = do
+        m <- (wrap `catch` h)
+        maybe (return ()) (\msg -> assertBool msg False) m
+      where
+        wrap = do
+            let ef out = Just ("expected exception on bad tx, got: "
```

Comment:
Note we are handling both exceptions and `Left foo` (what it actually returns) here -- I could change that